about.html

```diff
@@ -15,16 +15,10 @@
 <link rel="icon" type="image/ico" href="img/favicon.ico"  />
 
 <script src="js/jquery-2.1.3.min.js" type="text/javascript"></script>
+<script src="js/handlebars.js" type="text/javascript"></script>
```

Comment:
I would prefer using a CDN rather than importing the javascript into source here.